### PR TITLE
[TASK] !!! change TYPO3 Requirements

### PR DESCRIPTION
**Pull request by achimfritz**

### CODE REVIEW

**Review by achimfritz — COMMENTED** [review]
👍

File: Classes/Controller/StatisticsController.php [hunk 1]
```diff
@@ -29,68 +31,27 @@ class StatisticsController implements RequestHandlerInterface
...
-        LanguageService $languageService,
-        StandaloneView $view,
-        SiteFinder $siteFinder,
-        PageRepository $pageRepository
+        protected ModuleTemplateFactory $moduleTemplateFactory,
+        protected UriBuilder $uriBuilder,
+        protected SiteFinder $siteFinder,
+        protected PageRepository $pageRepository,
+        protected StandaloneView $view,
+        protected LanguageServiceFactory $languageServiceFactory
```
Comment (ervaude) [inline Classes/Controller/StatisticsController.php:51]: We should require PHP 8 in composer.json then

File: Configuration/Services.yaml [hunk 2]
```diff
@@ -8,4 +8,5 @@ services:
     resource: '../Classes/*'
 
   B13\Umami\Controller\StatisticsController:
+    tags: ['backend.controller']
     public: true
```
Comment (ervaude) [inline Configuration/Services.yaml]: We dont need public:true if we use the tag.

**Review by ervaude — CHANGES_REQUESTED** [review]

File: Configuration/Services.yaml (hunk 2 above)
Comment (achimfritz) [reply Configuration/Services.yaml]: done

**Review by ervaude — APPROVED** [review]

**Pull request merged**